### DISCUSSION TITLE
it is weird when visualizing DEGs using dotplot

### DISCUSSION THREAD

**Discussion by xiaowenchenjax** [satijalab/seurat#5910] [General]
Dear Author,
I used function "FindMarkers()" to identify DEGs in Old sample vs Young sample. I used the dot plot to visualize some interested genes, labeled DEGs in green and orange color. please see the following figure. 
<img width="655" alt="Screen Shot 2022-05-03 at 9 34 58 AM" src="https://user-images.githubusercontent.com/30912295/166463476-b67343e2-db65-46b1-9649-c9702629bdc8.png">

gene 'Fli1' is not DEG, but its average expression value distribution in old and young is similar with DEG 'Stat5b'. It looks weird. Could you help me to explain that?
Thanks

**Comment by timoast** [comment]
Please see https://github.com/satijalab/seurat/issues/5742

**Comment by xiaowenchenjax** [comment]
Hi @timoast 
I see, Thanks. Shall I try set scale = False?

**Reply by timoast** [reply]
sure
